spec/test-utils/test_indexeddb_cryptostore_dump/unverified/README.md

```diff
@@ -0,0 +1,3 @@
+The corresponding pickle key is `WVllQb4Lk/WwP4Q7iBfeTUHpgydZm9YqXI1B5bTvnIM`.
+userId: `@untrusted:localhost`
+deviceId: `VJPSPVPWZT`
```

Comment:
please could you expand this to document the directory properly.

Reply:
Done